packages/apps/spaces/app/organization/[id]/src/components/Tabs/panels/AboutPanel/AboutPanel.tsx

```diff
@@ -110,7 +104,7 @@ export const AboutPanel = () => {
             ) {
               return next;
             }
-            throttledMutatOrganization(state.values, {
+            debouncedMutateOrganization(state.values, {
```

Comment:
apparently it's worse then debounce because it will setDefaultValues every `300` ms and will reset whatever change you did after the last update fired and the next timeout. It happens also with debounce less often.

Reply:
does that restores also the issues we've had with the debouce that were fixed with throttle?